src/scenes/Main/Main.view.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import classNames from 'classnames';
+
+import { Title } from '@game/components';
+import * as Game from './components';
+
+import { IOptions } from '@game/types';
+
+import './style.scss';
+
+interface IMain {
+    options: IOptions;
+    winner: boolean;
+    turn: boolean;
+    field: string[];
+    children?: React.ReactNode;
```

Comment:
it should be child as far as it's singular

Reply:
React passes it as children not as child